packages/web-components/src/_docs/developer/quick-start.stories.mdx

```diff
@@ -0,0 +1,54 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Concepts/Developer/Quick Start" />
+
+## Install
+
+Fluent UI should be installed as a `dependency` of your app.
+
+**Yarn**
+
+```sh
+yarn add @fluentui/web-components
+```
+
+**NPM**
+
+```sh
+npm i @fluentui/web-components
```

Comment:
```suggestion
npm i --save @fluentui/web-components
```

Reply:
Do we need this in modern versions of NPM (greater than 5.0.0)?

Reply:
Good point, I'm out of date!